Content.Shared/GameObjects/Components/Chemistry/SolutionComponent.cs

```diff
@@ -21,6 +21,11 @@ public class SolutionComponent : Component
         protected int _maxVolume;
         private SolutionCaps _capabilities;
 
+        /// <summary>
+        /// Triggered when the solution contents change.
+        /// </summary>
+        public event EventHandler SolutionChanged;
```

Comment:
Just make this an `Action` event. The default `object sender, EventArgs args` arguments from `EventHandler` are nigh useless.

Reply:
Yeah you're right, that's better. I missed `Action` in the docs. Done.